README.md

```diff
@@ -47,3 +51,9 @@ cat <<EOF > server.json
 EOF
 pm2 start server.json
 ```
+
+## TESTING
```

Comment:
I think a quick update to this branch would be best actually. Kill two birds with one stone as they say. :bird: :moyai: